File: api/core/src/main/kotlin/domain/Ucenje.kt
package domain

import base.Id
import kotlinx.datetime.LocalDate
import kotlinx.serialization.Serializable

@Serializable
data class Ucenje(
    override var _id: Id<Ucenje> = Id(),
    var oseba_ucenec_id: Id<Oseba>,
    var oseba_ucitelj_id: Id<Oseba>,
    var ustvarjeno: LocalDate
) : Entiteta<Ucenje>

File: app/src/app/core/services/api/models/ucenje.ts
import { Id } from './id'
import { Oseba } from './oseba'
export interface Ucenje {
	'_id': Id<Ucenje>
	oseba_ucenec_id: Id<Oseba>
	oseba_ucitelj_id: Id<Oseba>
	razred: string
	ustvarjeno: string
}